commit 4c58eb52c413f34aa1f085d017b736afa140b227
Author: Andrew Price <anprice@redhat.com>
Date:   Wed Feb 22 12:05:03 2017 -0500

    gfs2: Add missing rcu locking for glock lookup
    
    commit f38e5fb95a1f8feda88531eedc98f69b24748712 upstream.
    
    We must hold the rcu read lock across looking up glocks and trying to
    bump their refcount to prevent the glocks from being freed in between.
    
    Signed-off-by: Andrew Price <anprice@redhat.com>
    Signed-off-by: Andreas Gruenbacher <agruenba@redhat.com>
    Signed-off-by: Bob Peterson <rpeterso@redhat.com>
    Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>

Index: linuxtest/dummy/rpi_1326_4c58eb52c413f34aa1f085d017b736afa140b227.txt
===================================================================
--- /dev/null
+++ linuxtest/dummy/rpi_1326_4c58eb52c413f34aa1f085d017b736afa140b227.txt
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
